apps/remix-ide-e2e/src/tests/recorder.test.ts

```diff
@@ -73,11 +75,16 @@ module.exports = {
       .pause(1000)
       .createContract('')
       .click('i.savetransaction')
-      .modalFooterOKClick()
+      .waitForElementVisible('[data-id="udappNotify-modal-footer-ok-react"]')
+      .execute(function () {
+        const modalOk = document.querySelector('[data-id="udappNotify-modal-footer-ok-react"]') as any
+
+        modalOk.click()
+      })
       .click('*[data-id="deployAndRunClearInstances"]') // clear udapp
       .click('*[data-id="terminalClearConsole"]') // clear terminal
-      .click('#runTabView .runtransaction')
-      .clickInstance(1)
+      .click('[data-id="runtransaction"]')
+      .clickInstance(2)
```

Comment:
the input param for `clickInstance` doesn't seem to have changed. why do you need to put `2` instead of `1`

Reply:
@yann300 The reason is that previously we increment the index passed into clickInstance command. Now we do not so the increased index is passed in now. https://github.com/ethereum/remix-project/pull/1850/files/58d1638b115c5102d126aabb4fb33af1722a7eba#diff-9d4fe3681c2fef5cd6e7c9fc9f52c230952964cad3b68cabbd938395c9bd1716L6

Reply:
yes, the reason i ask that is that I see the use of  `clickInstance` hasn't been updated in some other places although I supposed it should have.